B/Bzip2/build_tarballs.jl

```diff
@@ -25,35 +25,34 @@ make CFLAGS="${CFLAGS}" PREFIX=${prefix} install
 
 # Build dynamic library
 if [[ "${target}" == *-darwin* ]]; then
-    $CC -shared -current_version 1.0.6 -compatibility_version 1.0 -o libbz2.1.0.6.dylib $LDFLAGS $OBJS
-    ln -s libbz2.1.0.6.dylib libbz2.1.0.dylib
-    ln -s libbz2.1.0.6.dylib libbz2.1.dylib
-    ln -s libbz2.1.0.6.dylib libbz2.dylib
-    mv libbz2*.dylib ${prefix}/lib/
+    cc -shared -current_version "${VERSION}" -compatibility_version 1.0 -o "libbz2.${VERSION}.dylib" ${OBJS}
+    ln -s libbz2.${VERSION}.dylib libbz2.1.0.dylib
+    ln -s libbz2.${VERSION}.dylib libbz2.1.dylib
+    ln -s libbz2.${VERSION}.dylib libbz2.dylib
 elif [[ "${target}" == *-mingw* ]]; then
-    $CC -shared -o libbz2-1.dll $LDFLAGS $OBJS
+    cc -shared -o libbz2-1.dll ${OBJS}
     ln -s libbz2-1.dll libbz2.dll
-    mv libbz2*.dll ${prefix}/bin/
 else
-    $CC -shared -Wl,-soname -Wl,libbz2.so.1.0 -o libbz2.so.1.0.6 $LDFLAGS $OBJS
-    ln -s libbz2.so.1.0.6 libbz2.so.1.0
-    ln -s libbz2.so.1.0.6 libbz2.so.1
-    ln -s libbz2.so.1.0.6 libbz2.so
-    mv libbz2.so* ${prefix}/lib/
+    cc -shared -Wl,-soname -Wl,libbz2.so.1.0 -o "libbz2.so.${VERSION}" ${OBJS}
+    ln -s "libbz2.so.${VERSION}" libbz2.so.1.0
+    ln -s "libbz2.so.${VERSION}" libbz2.so.1
+    ln -s "libbz2.so.${VERSION}" libbz2.so
 fi
```

Comment:
For future reference, the makefile `Makefile-libbz2_so` might be able to replace most of this (but some symlinks are probably still missing)